controller/clusters.go

```diff
@@ -36,15 +36,16 @@ func (c *ClustersController) Show(ctx *app.ShowClustersContext) error {
 		return app.JSONErrorResponse(ctx, errors.NewUnauthorizedError("unauthorized access to cluster info"))
 	}
 	var data []*app.ClusterData
-	for _, clusterConfig := range c.config.GetOSOClusters() {
+	for _, configCluster := range c.config.GetClusters() {
 		cluster := &app.ClusterData{
```

Comment:
same here with the `cluster` var name which may collide with an existing pkg

Reply:
addressed in ec3061b